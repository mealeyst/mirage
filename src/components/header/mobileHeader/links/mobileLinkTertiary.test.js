import React from 'react'
import 'jest-styled-components'

import { MobileLinkTertiary } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) MobileLinkTertiary', () => {
  const createMobileLinkTertiary = (props) => {
    return shallowWithTheme(<MobileLinkTertiary {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createMobileLinkTertiary())
    .toMatchSnapshot()
  })
})
