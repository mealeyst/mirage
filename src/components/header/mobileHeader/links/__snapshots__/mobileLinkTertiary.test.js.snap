// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) MobileLinkTertiary matching the snapshot 1`] = `
.c0 {
  height: 3.6rem;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-left: 0.5rem;
  padding-right: 0.5rem;
  background-color: transparent;
}

.c0:active,
.c0:focus,
.c0:hover,
.c0:visited {
  color: #0073D1;
}

<Styled(Styled(Component))
  background="transparent"
  className="c0"
  fontSize="1.6rem"
  fontStyle="italic"
  light={true}
  underline={false}
/>
`;
